Description: Patch is applied because autotools are not used by upstream.
 Remove compiler optimizations.
Author: Jaromír Mikeš <mira.mikes@seznam.cz>
Forwarded: no

--- a/source/Makefile
+++ b/source/Makefile
@@ -33,8 +33,7 @@
 
 CPPFLAGS += -I. -D_REENTRANT -D_POSIX_PTHREAD_SEMANTICS
 CPPFLAGS += -DENABLE_VECTOR_MODE 
-CXXFLAGS += -fPIC -Wall -ffast-math -funroll-loops -O2
-CXXFLAGS += -march=native
+CXXFLAGS += -fPIC -Wall -ffast-math -funroll-loops
 LDLFAGS += 
 LDLIBS +=
 
@@ -56,7 +55,7 @@
 	install -d $(DESTDIR)$(LIBDIR)
 	install -m 644 $(ZITA-CONVOLVER_H) $(DESTDIR)$(INCDIR)
 	install -m 755 $(ZITA-CONVOLVER_MIN) $(DESTDIR)$(LIBDIR)
-	ldconfig
+	/sbin/ldconfig -n $(DESTDIR)$(PREFIX)/$(LIBDIR)
 	ln -sf $(ZITA-CONVOLVER_MIN) $(DESTDIR)$(LIBDIR)/$(ZITA-CONVOLVER_SO)
 
 uninstall:
